src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/ForgotPasswordLinkSent.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿<div class="nhsuk-width-container">
+    <main class="nhsuk-main-wrapper">
+        <div data-test-id="login-page">
+            <div class="nhsuk-grid-row">
+                <div class="nhsuk-grid-column-two-thirds">
+
+                    <div class="bc-c-data-link">
```

Comment:
I'm not sure this class exists? `bc-c-data-link`

Reply:
we have the class `nhsuk-back-link` on the div above and `nhsuk-back-link__link` on the a tag

Reply:
It does exist, but I will switch to the NHS ones for consistency